### ISSUE TITLE
Listview is deprecated

### ISSUE THREAD

**Issue by myunusss** [mrlaessig/react-native-autocomplete-input#111]
everything works fine 👍, but give me a warning like this : 

" ListView is deprecated and will be removed in a future release. "

is it possible to replace with FlatList ?

thank you

**Comment by amrita019** [comment]
@myunusss Any update on this?

**Comment by kristoferpelchat** [comment]
@mrlaessig Hi :) Do you plan on moving to FlatList above as @myunusss suggests? I would hate to see ListView go away and this AWESOME library is still using it. Thoughts?

**Comment by msqar** [comment]
Damn i was hoping for this to be solved. I want this to be migrated to FlatList too... i might create a PR for this, though.

**Comment by Javier-Rotelli** [comment]
welp, there are at least 2 PR's fixing this (#65 and #83) =/
but no answer from @mrlaessig

**Comment by ConnectedReasoning** [comment]
anybody want to fork this?

**Comment by Javier-Rotelli** [comment]
hi @ConnectedReasoning, I'm intending to maintain this repo. however as soon as i got permissions, life appeared. i'll be merging PR's and doing housekeeping on the weekend

**Comment by ConnectedReasoning** [comment]
@Javier-Rotelli Thats awesome, thanks. I'll keep a lookout and can help you test. I was looking over your changes and so far so good. Looks like users need to provide a couple more props?

**Comment by ConnectedReasoning** [comment]
@Javier-Rotelli this is an example of a good react autocomplete implementation. [React Materialize Autocomplete](http://react-materialize.github.io/react-materialize/?path=/story/javascript-autocomplete--default.)

I would like to see our react-native component here implement these features and behaviors minus the materialize specifics of course. Let me know what you think.

**Comment by ConnectedReasoning** [comment]
Happy to see the updates! I will try them out in my app today

**Comment by ConnectedReasoning** [comment]
Ok, npm install brought in the old version. Maybe something has to be updated at npm? I copied and pasted the index.js into my node modules and I am no longer getting the listview error

**Comment by Javier-Rotelli** [comment]
Hi @ConnectedReasoning  good to hear that, i didn't made the release yet.
it worked without changing anything on your code?. I haven't had the time to test this properly, and i don't want to ship breaking changes on a minor or patch release

**Comment by ConnectedReasoning** [comment]
I had removed autocomplete from my code as it was giving me the warnings. I did not have to change anything out of the ordinary to get this to work after I copied and pasted the latest index.js to my node_modules auto complete installation.  I was even able to apply typeahead filtering within my own code. [GithubGist](https://gist.github.com/ConnectedReasoning/e258793d10ea31d10a32db88dcd83977)
